spec/controllers/clients_controller_spec.rb

```diff
@@ -119,6 +120,170 @@
           )
         }
       end
+
+      ###### Test scenario #4 #######
+      context 'Customer has three passports, same Client ID' do
+        it {
```

Comment:
the `it` block should have a string describing what the test checks, e.g `it "is eligible"`